lib/routes/sign.js

```diff
@@ -45,80 +49,93 @@ module.exports = function (log, isA, error, signer, db, domain, metricsContext)
           return reply(error.unverifiedAccount())
         }
 
-        if (publicKey.algorithm === 'RS') {
```

Comment:
Indentation changed as this stuffed moved inside a `.then()`. The only meaningful difference should be the new call to `db.createDevice`.